lib/Sources/WebView/Resources/Generated/BluetoothPolyfill.js

```diff
@@ -17,6 +17,9 @@ const bluetoothRequest = function (action, data) {
         data: data
     }).catch(rethrowAsDOMException);
 };
+const appLog = function (msg) {
```

Comment:
I'm assuming I don't understand something about js/ts, but why is this all redefined?

Reply:
This is the compiled artifact. The `ts` is the source, the `js` is the "binary".